<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMove</class>
 <widget class="QDialog" name="DialogMove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Additional info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresources.qrc">
    <normaloff>:/png/png/questionmark.png</normaloff>:/png/png/questionmark.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-3600.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3600.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter time interval: </string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="starttimeEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="maximumTime">
    <time>
     <hour>9</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss.zzz</string>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="text_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="endtimeEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumTime">
    <time>
     <hour>9</hour>
     <minute>59</minute>
     <second>59</second>
    </time>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::HourSection</enum>
   </property>
   <property name="displayFormat">
    <string>HH:mm:ss.zzz</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="text_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Until:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Increase</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Decrease</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="myresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogMove</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogMove</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
